SodaInventory/ClientApp/src/pages/InventoryForm.js

```diff
@@ -54,7 +53,9 @@ class InventoryForm extends Component {
 						"itemQuantityId": quant.itemQuantityId,
 						"itemId": quant.itemId,
 						"storeId": quant.storeId,
-						"amount": parseInt(quantities[quant.itemId])
+						"amount": parseInt(quantities[quant.itemId]),
+						"moderateLevel": quant.moderateLevel,
+						"urgentLevel": quant.urgentLevel,
```

Comment:
Is there a reason we are adding this data to the inventory form?  Is it needed?

Reply:
Yes, It addresses an issue with saving the items. Without this it removed the moderate and urgent levels from the database.